### ISSUE TITLE
Main camera sensor not working

### ISSUE THREAD

**Issue by Iliasnoir** [PixelExperience/android-issues#3101]
## Build date

SQ3A.220605.009.A1
PixelExperience_Plus_dipper-12.1-20220629-1743-OFFICIAL

## Expected Behavior

Main camera sensor working

## Current Behavior

Only Secondary camera sensor working on all apps

## Possible Solution

Edit here

## Steps to Reproduce

1. Open any camera app
2.
3.
4.

/device dipper
/version twelve_plus

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by GiaSen** [comment]
any log? any step to reproduce? doesn't look like anyone got ever got this issue

**Comment by Iliasnoir** [comment]
I got the ROM in the last few days, and it was like that from the
beginning. I noticed it trying to take some videos from my car and I saw
that the phone was using the bottom camera and not the top one. Even when I
zoom or etc only the bottom camera is being used. I can't seem to find the
hardware test menu to test if the camera is working fine. Maybe it's
something I'm the settings that I can't find.
